sys/kern/kern_exit.c

```diff
@@ -205,12 +205,13 @@ exit_onexit(struct proc *p)
 /*
  * exit -- death of process.
  */
-void
-sys_sys_exit(struct thread *td, struct sys_exit_args *uap)
+int
+sys_exit(struct thread *td, struct exit_args *uap)
```

Comment:
This is a FreeBSD wart, but I've not chased down why we did it. All historic unixes through 4.4 used the bare exit()